src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -138,11 +144,11 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     }
   }
 
-  const setIsActive = () => {
+  const setIsActive = (text: string | undefined | null = '') => {
     // Tells application state that a plugin field is in focus
     setUpdateTrackerState({
       type: 'setElementEntered',
-      textValue: value || '',
+      textValue: text || '',
     })
```

Comment:
Hmm, i just realised this breaks validity check (if values is not changed). i.e start test review application, fill the first page, refresh the page, go into a field (in and out), try to press review and summary, nothing happens.

I wont' go into details but data is not changed but validity checks thinks it should have changed.

`const setIsActive = (text: string | undefined | null = '') => { ` -> `const setIsActive = () => { ` 
`textValue: text || '',` => `textValue: currentResponse?.text || '',`

Reply:
Cool, thanks 35cf5a02